projects/js-packages/connection/components/disconnect-dialog/steps/step-disconnect-confirm.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal Dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { DecorativeCard } from '@automattic/jetpack-components';
+import disconnectImage from '../images/disconnect-confirm.jpg';
```

Comment:
I'm not very fond of storing images in the RNA package.
What would you say about other options?

1. Store images in the consuming plugin, and pass image paths as properties: [example in Jetpack](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/_inc/client/main.jsx#L269).
However, this might be a bit tedious due to props drilling.

2. It looks like the image can be set via CSS [here](https://github.com/Automattic/jetpack/blob/4eafc91d1208c4a71fcd55e71d93e8854abf96e0/projects/js-packages/components/components/decorative-card/index.jsx#L45).
May be we should get rid of the image property in the `imageUrl` property in the `DecorativeCard` component, and let consumers set the background in their CSS files?
It would need to be explained in the docs for the component, but I think this approach would also work.

Reply:
@jboland88 pointed out that the discussion already happened in Slack: p1634910123095200-slack-CBG1CP4EN